extensionsGallery.json

```diff
@@ -3074,7 +3074,47 @@
 					],
 					"statistics": [],
 					"flags": ""
-				}
+				},
+                {
+                  "extensionId": "62",
+                  "extensionName": "qpi",
+                  "displayName": "Query Performance Insight",
+                  "shortDescription": "ADS UI for the latest version of QPI library",
+                  "publisher": {
+                    "displayName":"Jovan Popovic",
+                    "publisherId": "jocapc",
+                    "publisherName":"jocapc"
+                  },
+                  "versions": [
+                    {
+                      "version": "0.1.6",
+                      "lastUpdated": "03/08/2020",
+                      "assetUri": "",
+                      "fallbackAssetUri": "fallbackAssetUri",
+                      "files": [
+                        {
+                          "assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
+                          "source": "https://github.com/JocaPC/AzureDataStudio-QPI/releases/download/stable/qpi-0.1.6.vsix"
+                        },
+                      {
+                        "assetType": "Microsoft.VisualStudio.Services.Content.Details",
+                      "source": "https://raw.githubusercontent.com/jocapc/AzureDataStudio-QPI/master/README.md"
```

Comment:
Please add more information to https://github.com/JocaPC/AzureDataStudio-QPI/blob/master/README.md. See https://raw.githubusercontent.com/microsoft/azuredatastudio/master/extensions/admin-tool-ext-win/README.md for an example. At the very least it should have a brief description of what the extension is and how to use it.

Reply:
Changed readme